src/server/src/test/java/io/cassandrareaper/acceptance/ReaperCassandraIT.java

```diff
@@ -39,7 +39,8 @@
 
 @RunWith(Cucumber.class)
 @CucumberOptions(
-    features = "classpath:io.cassandrareaper.acceptance/integration_reaper_functionality.feature"
+    features = "classpath:io.cassandrareaper.acceptance/integration_reaper_functionality.feature",
+    plugin = {"pretty"}
     )
 public final class ReaperCassandraIT {
```

Comment:
let's make this non-final and create empty subclasses that we can put the incremental or not incremental annotations on. same applies for all the *IT.java